zerver/management/commands/send_webhook_fixture_message.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import absolute_import
+from __future__ import print_function
+
+import os
+import ujson
+from optparse import make_option
+
+from django.test import Client
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help = """Create webhook message based on given fixture"""
+
+    option_list = BaseCommand.option_list + (
+        make_option('-f', '--fixture',
+                    dest='fixture',
+                    type='str',
+                    help='A path to fixture you\'d like to send'),
+        make_option('-u', '--url',
+                    dest='url',
+                    type='str',
+                    help='An address where data will be processed'),
```

Comment:
Probably the help here should be more clear that this is the URL on your Zulip server that you want to post the fixture to